packages/warriorjs-core/src/Unit.test.js

```diff
@@ -206,12 +206,19 @@ describe('Unit', () => {
       receiver.say = jest.fn();
     });
 
-    test('earns points equal to max health when killing unit', () => {
+    test('earns points equal to max health when killing unit without reward', () => {
       unit.earnPoints = jest.fn();
       unit.damage(receiver, 5);
       expect(unit.earnPoints).toHaveBeenCalledWith(5);
     });
 
+    test('earns points equal to reward when killing unit with reward', () => {
```

Comment:
@RascalTwo Assigning the reward to max health in the constructor makes this test obsolete. Instead, we should [pass a reward when instantiating the unit for tests](https://github.com/olistic/warriorjs/pull/67/files#diff-dc5de3455140fdf7cf88e9817a2134fbR11), and add a test to check that reward is assigned to max health when omitted.